### DISCUSSION TITLE
Fuel flow and level sensors for turbine and internal combustion planes

### DISCUSSION THREAD

**Discussion by pnduffy** [iNavFlight/inav#7407] [Q&A]
I'm using INav for my new projects, but it would be a nice feature if INav had support for fuel flow and level sensors like Ardupilot.  I building a turbine jet, and I can't find any equivalent feature in INav.  

https://ardupilot.org/plane/docs/common-fuel-sensors.html

**Comment by DjonnyKo** [comment]
Hi
Did you manage to connect the turbine to INAV?

**Comment by pnduffy** [comment]
No, but I designed a fuel flow monitor externally connected to INav.  I never got the turbine itself connected.

-Patrick

________________________________
